### PR TITLE
[13.0] EDI Add crons to automate record status

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: edi/models/edi_backend.py [hunk 1]
```diff
@@ -198,34 +206,88 @@ def _exchange_output_check(self, exchange_record):
...
+    def _output_new_records_domain(self):
+        return [
+            ("type_id.exchange_file_auto_generate", "=", True),
+            ("type_id.direction", "=", "output"),
+            ("edi_exchange_state", "=", "new"),
+            ("exchange_file", "=", False),
+        ]
+
+    def _output_pending_records_domain(self):
+        states = ("output_pending", "output_sent", "output_sent_and_error")
```
Comment (etobella) [inline edi/models/edi_backend.py:279]: Why output sent should be pending?

I would also add a check on the type in order to proceed with this. What do you think?

Comment (simahawk) [reply edi/models/edi_backend.py:279]: because `_check_output_exchange_sync` takes care of sending them as well

Comment (simahawk) [reply edi/models/edi_backend.py:279]: > I would also add a check on the type in order to proceed with this.

what do you mean? The type's direction is "output", what else?

Comment (etobella) [reply edi/models/edi_backend.py:279]: Maybe the output will be sent on user decision, not by a cron

It is just an idea (similar to output generation??)

Comment (simahawk) [reply edi/models/edi_backend.py:279]: hmm yeah, why not... 
I could also split sending from sent here

**Review by etobella — APPROVED** [review]

File: edi/models/edi_backend.py (hunk 1 above)
Comment (etobella) [reply edi/models/edi_backend.py:279]: We can keep it for version two, so, don't worry :smile:

**Comment by simahawk** [comment]
@etobella if it's ok for you I fast track this as it's the base for the next round.

**Pull request merged**